utils/log.go

```diff
@@ -79,13 +78,12 @@ func init() {
 }
 
 func readLoggingEnv() {
-	env := os.Getenv(logEnv)
-	if env == "" {
-		return
+	switch os.Getenv(logEnv) {
```

Comment:
I'd like to be backwards compatible for the time being (and emit a noticable warning to switch to the new strings). So, please add parsing with the old numeric log levels.